recipe_Menager.py

```diff
@@ -0,0 +1,42 @@
+class RecipeManager:
+    def __init__(self):
+        self.recipes = []
+
+    def add_recipe(self, recipe):
+        self.recipes.append(recipe)
+
+    def update_recipe(self, recipe_index, new_recipe):
+        self.recipes[recipe_index] = new_recipe
+      #on this point I am confused, because I can see 2 meanings:
+      # 1. change older recipe for new one and I had choosen this version
+      # 2. change the integrent in recipe
+
+    def delete_recipe(self, recipe_index):
+        self.recipes.pop(recipe_index)
+
+    def display_recipes(self):
+        for recipe in self.recipes:
+            print(recipe)
+
+
+##
+##tests
+##
+recipe_manager = RecipeManager()
+
+recipe1 = {'name': 'Spaghetti Carbonara', 'ingredients': ['spaghetti', 'bacon', 'eggs']}
```

Comment:
All tests really need to be added into a separate file contained within a "tests" directory, also a consideration or quantities needs to be considered - I might have a crack at this myself.